lib/components/narrative/narrative.css

```diff
@@ -195,7 +195,7 @@
 
 .otp .tabbed-itineraries .tab-button .details {
   font-size: 12px;
-  color: gray;
+  color: #685c5c; /* from "gray" to meet WCAG */
```

Comment:
We probably don't need a comment about the previous value here.

Reply:
Removed in 6188d3ac981359220491cc83995fe7f351d9efb2